### PR TITLE
Add support for WPML (plugin / add-ons)

### PR DESCRIPTION
**Pull request by mcaskill**
## Checklist

- [x] I've read the [Contributing page](https://github.com/junaidbhura/composer-wp-pro-plugins/blob/master/CONTRIBUTING.md).
- [x] Issue: #43
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [X] My code has proper inline documentation.

## Description

Supersedes PR #42 to add support for WPML and add-ons.

This pull request includes the suggestions (coding style and typo fixes) from the former as well latest changes to the `master` branch.

## How has this been tested?

I'm testing this on a client project that uses ACF Pro, ACF Extended Pro, Ninja Forms, PublishPress Pro, and WPML.

### CODE REVIEW

File: README.md
Comment (junaidbhura) [inline README.md:250]: @mcaskill Does it make sense to shorten this? What is the name of the plugin that gets downloaded?

**Review by junaidbhura — CHANGES_REQUESTED** [review]

File: README.md
Comment (mcaskill) [reply README.md:250]: The ZIP from wpml.org uses the directory name `sitepress-multilingual-cms`, its text domain is also that, and internally appears to expect the directory to be named that (which requires end-users to change its name via `extra.installer-name`).

**Review by junaidbhura — APPROVED** [review]

**Comment by mcaskill** [comment]
~~Since #52 has also been approved, do we merge that one first and afterwards I update the WPML pull request accordingly?~~

Do we merge #52 first?

**Comment by junaidbhura** [comment]
@mcaskill #52 isn't quite approved yet. But yes, lets merge this one after that one is merged in!

**Comment by junaidbhura** [comment]
@mcaskill #52 is merged in!

**Comment by junaidbhura** [comment]
@mcaskill Lemme know when you are happy to merge this in

**Comment by mcaskill** [comment]
Should be good to merge now.

I had forgotten about the indentation of the `Wpml.php` file. Which made me notice that `RemoteFilesystem.php` also has mixed indentation but falls outside the scope of this PR.

**Comment by mcaskill** [comment]
@Spidlace Thank you so much for your help!

**Comment by junaidbhura** [comment]
@mcaskill Thanks, as always, for your hard work on this! A new version has now been released `1.7.0` 🚀

**Comment by mcaskill** [comment]
@junaidbhura You're welcome! This has been a very useful Composer plugin over the years.

By the way, could you include @Spidlace in the release for 1.7.0?
They did the bulk of the work on getting WPML supported.

**Comment by junaidbhura** [comment]
Good call @mcaskill I've updated the release, thank you @Spidlace!

**Pull request merged**
